e2e/packages/sync-test/createStoreSync.test.ts

```diff
@@ -0,0 +1,43 @@
+import { describe, expect, it } from "vitest";
+import { rpcHttpUrl } from "./setup/constants";
+import { deployContracts } from "./setup";
+import { createAsyncErrorHandler } from "./asyncErrors";
+import { createWorld, getComponentValueStrict } from "@latticexyz/recs";
+import { singletonEntity, syncToRecs } from "@latticexyz/store-sync/recs";
+import mudConfig from "../contracts/mud.config";
+import { transportObserver } from "@latticexyz/common";
+import { ClientConfig, createPublicClient, http } from "viem";
+import { getNetworkConfig } from "../client-vanilla/src/mud/getNetworkConfig";
+
+const address = "0xad97505a508daf984fe60302109e0115e544b267";
+
+describe("createStoreSync", () => {
```

Comment:
I assume this is here because we haven't created all the scaffolding (anvil, database, etc.) in store-sync tests yet?

Reply:
Yes - `store-sync` doesn't have a contracts folder, `execa`, etc. I kind of like that it is more pure but we do risk `e2e` getting bloated with tests that are not fully "end to end"